src/elements/List/List.js

```diff
@@ -1,32 +1,44 @@
-import React, { Component, PropTypes } from 'react'
-import classNames from 'classnames'
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
 import META from '../../utils/Meta'
+import {getUnhandledProps} from '../../utils/propUtils'
 import ListItem from './ListItem'
 
-export default class List extends Component {
-  static propTypes = {
-    children: PropTypes.node,
-    className: PropTypes.string,
-  }
-
-  static _meta = {
-    library: META.library.semanticUI,
-    name: 'List',
-    type: META.type.element,
-  }
-
-  static Item = ListItem
-
-  render() {
-    const classes = classNames(
-      'ui',
-      this.props.className,
-      'list'
-    )
-    return (
-      <div {...this.props} className={classes}>
-        {this.props.children}
-      </div>
-    )
-  }
+function _List(props) {
+  const {
+    children, className,
+    } = props
+
+  const classes = cx(
+    'ui',
+    'list',
+    className
+  )
+
+  const ListComponent = 'div'
+  const rest = getUnhandledProps(List, props)
+
+  return (
+    <ListComponent className={classes} {...rest}>
```

Comment:
Since the `ListComponent` never change, let's use a `div` inline here instead of creating the assignment:

```diff
-const ListComponent = 'div'
const rest = getUnhandledProps(List, props)

return (
-  <ListComponent className={classes} {...rest}>
+  <div className={classes} {...rest}>
    {children}
-  </ListComponent>
+  </div>
```